druid.go

```diff
@@ -23,6 +24,7 @@ const (
 	defaultRetryWaitMin          = 100 * time.Millisecond
 	defaultRetryWaitMax          = 3 * time.Second
 	defaultRetryMax              = 5
+	defaultSkipTlsOption         = false
```

Comment:
Golang convention is to have acronyms uppercased, so defaultSkipTLSOption. related and interesting: https://talks.golang.org/2014/names.slide#1

Reply:
Actually, that's not even needed now.

Reply:
This has been removed